internal/constants.go

```diff
@@ -6,5 +6,5 @@ const (
 
 	// JSONSchemaVersion is the current schema version output by the JSON encoder
 	// This is roughly following the "SchemaVer" guidelines for versioning the JSON schema. Please see schema/json/README.md for details on how to increment.
-	JSONSchemaVersion = "3.3.2"
+	JSONSchemaVersion = "4.0.0"
```

Comment:
Is this actually a _breaking_ change? I thought we were just adding _more_ information, so this would just be `3.3.3`?

Reply:
We also modified the Conan datatype to be Ref instead of name/version which is breaking

Reply:
![Screen Shot 2022-09-29 at 2 13 31 PM](https://user-images.githubusercontent.com/32073428/193110912-db00a55a-5023-4591-8b86-4a9ab75c6bb4.png)